src/App.css

```diff
@@ -553,4 +553,85 @@ a {
     border: 0;
   }
 }
-/* single page css ends here */
\ No newline at end of file
+/* single page css ends here */
+/* play list css starts here */
+.playlist-card-wrapper {
+  border: 1px solid var(--div-background-color);
+  padding: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  gap: 0.9rem;
+}
+.playlist-main-wrapper {
+  width: 90%;
+  margin:auto;
+  display:flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+.playlist-icon {
+  padding:0 0.3rem 0 0;
+}
+.myplaylist-create-wrapper {
+  display:flex;
+  justify-content: space-between;
+}
+.playlist-card-header {
+  display:flex;
+  justify-content: space-between;
+  align-items:center;
+}
+.playlist-cards-wrapper {
+  display:flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+.open-playlistBtn {
+  align-self: center;
+}
```

Comment:
Try using the naming convention whenever there are multiple words in your class name -> eg. btn-open-playlist

Reply:
Sure